README.md

```diff
@@ -2,36 +2,125 @@
 
 [![Build](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/ci.yml)
 [![Trivy](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/trivy-scan.yml)
-[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/build-with-bal-test-graalvm.yml)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-notion/actions/workflows/build-with-bal-test-native.yml)
```

Comment:
Is this workflow available?

Reply:
I actually had initially started with what I found [here](https://github.com/ballerina-platform/ballerina-library/blob/main/library-templates/generated-connector-template/files/README.md) will change this back.